### PR TITLE
docs: note that cannot reduce Region size

### PR DESCRIPTION
**Pull request by jennifertrin**
As per [this forum post](https://forum.dfinity.org/t/shrink-region-by-using-region-grow-region-0/29672), note that shrinking or reducing a Region size is currently not possible

### CODE REVIEW

File: doc/md/base/Region.md [hunk 1]
```diff
@@ -109,6 +109,8 @@ Every new page is zero-initialized, containing byte 0x00 at every offset.
 Function `grow` is capped by a soft limit on `size` controlled by compile-time flag
  `--max-stable-pages <n>` (the default is 65536, or 4GiB).
 
+There is currently no way to shrink the Region size.
+
```
Comment (crusso) [inline doc/md/base/Region.md:113]:
```suggestion
There is currently no way to shrink the Region size.

```

Comment (crusso) [reply doc/md/base/Region.md:113]: This is an auto generated file - I'll instead modify the source herehttps://github.com/dfinity/motoko-base/pull/632

Comment (crusso) [inline doc/md/base/Region.md:113]:
```suggestion
```
Reverting change to copy

**Review by crusso — APPROVED** [review]

File: doc/md/stable-regions.md [hunk 2]
```diff
@@ -49,6 +49,7 @@ module {
   // Each page is 64KiB (65536 bytes).
   // Returns previous `size` when able to grow.
   // Returns `0xFFFF_FFFF_FFFF_FFFF` if remaining pages of physical stable memory insufficient.
+  // Please note that there is currently no way to shrink the Region size.
```
Comment (crusso) [inline doc/md/stable-regions.md:52]:
```suggestion
  // Please note that there is no way to shrink the size of a region.
```

**Review by crusso — APPROVED** [review]

**Comment by jennifertrin** [comment]
Thank you, @crusso for the help here!

**Comment by crusso** [comment]
@jennifertrin thanks for taking the time to open a PR!

**Pull request merged**